pkg/run/run.go

```diff
@@ -29,19 +29,33 @@ const (
 	appProbeThreshold = 10
 )
 
+// Config for the Runtime
+type Config struct {
+	// RunTimeout is the total time to run
+	RunTimeout time.Duration
+
+	// WaitKubeletGC is the duration period to wait until the kubelet is garbage collected
+	WaitKubeletGC time.Duration
+
+	// DNSQueryForReadiness are the dns query to execute to ack the readiness
+	DNSQueryForReadiness []string
```

Comment:
Maybe `KubeletGCTimeout` and `ReadinessDNSQueries` or `DNSQueriesForReadiness` (it just needs to be plural since it's a slice)?